phpunit.el

```diff
@@ -140,9 +144,6 @@
   (defvar-local phpunit-root-directory nil
     "Directory path to execute PHPUnit.")
   (put 'phpunit-root-directory 'safe-local-variable #'stringp)
-  (defvar-local phpunit-args nil
-    "Argument to pass to phpunit command.")
-  (put 'phpunit-args 'safe-local-variable #'(lambda (v) (or (stringp v) (listp v))))
```

Comment:
This definition is important for [file/directory local variables.](https://www.gnu.org/software/emacs/manual/html_node/emacs/Directory-Variables.html)